Themes/default/languages/Admin.english.php

```diff
@@ -485,12 +485,12 @@
 $txt['gravatar_maxR'] = 'R rated (Restricted)';
 $txt['gravatar_maxX'] = 'X rated (Explicit)';
 $txt['gravatarDefault'] = 'Default image to show when an email address has no matching Gravatar ';
-$txt['gravatar_mm'] = 'A simple, cartoon-style silhouetted outline of a person';
-$txt['gravatar_identicon'] = 'A geometric pattern based on an email hash';
-$txt['gravatar_monsterid'] = 'A generated \'monster\' with different colors, faces, etc';
+$txt['gravatar_mm'] = 'Static, cartoon-style silhouetted outline of a person';
+$txt['gravatar_identicon'] = 'Generated geometric pattern';
+$txt['gravatar_monsterid'] = 'Generated \'monster\' with different colors, faces, etc';
```

Comment:
More clearer English?

Reply:
The reporter of the issue was confused about which gravatars was generated for every user and which ones only showed a static image, so thought it could be made more clear. But I am not a native english speaker, so feel free to propose changes if anything feels wrong.

Reply:
I can remove the change of these strings as well. They are not the important part of this PR.

Reply:
Yes please. I'm thinking we can just keep the first commit, since I definitely cannot come up with a better way to reword those.

Reply:
Ok, done